apps/admin-x-settings/src/components/settings/email/newsletters/NewsletterDetailModal.tsx

```diff
@@ -219,8 +219,9 @@ const Sidebar: React.FC<{
             return (
                 <TextField
                     error={Boolean(errors.sender_email)}
-                    hint={errors.sender_email || `If left empty, ${defaultEmailAddress} will be used`}
-                    rightPlaceholder={`@${sendingDomain(config)}`}
+                    hint={errors.sender_email}
+                    placeholder={sendingEmailUsername ? `@${sendingDomain(config)}` : `${defaultEmailAddress}` }
```

Comment:
This can get simplified to 
```suggestion
                    placeholder={defaultEmailAddress}
```
Because the placeholder won't be visible if `sendingEmailUsername` is set